itl/command-plugins.conf

```diff
@@ -275,15 +275,35 @@ object CheckCommand "ssl" {
 	command = [ PluginDir + "/check_tcp" ]
 
 	arguments = {
-		"-H" = "$ssl_address$"
-		"-p" = "$ssl_port$"
+		"-H" = {
+			value = "$ssl_address$"
+			description = "Host address"
+		}
+		"-p" = {
+			value = "$ssl_port$"
+			description ="TCP port (default: 443)"
+		}
 		"--ssl" = { }
-		"--timeout" = "$ssl_timeout$"
-		"-D" = "$ssl_cert_valid_days_warn$,$ssl_cert_valid_days_critical$"
+		"-t" = {
+			value = "$ssl_timeout$"
+			description = "Seconds before connection times out (default: 10)"
+		}
+		"-D" = {{
+			if (macro("$ssl_cert_valid_days_warn$")) {
+				if (macro("$ssl_cert_valid_days_critical$")) {
+					return macro("$ssl_cert_valid_days_warn$,$ssl_cert_valid_days_critical$")
+				} else {
+					return macro("$ssl_cert_valid_days_warn$")
+				}
+			}
```

Comment:
I'd use local variables here, and save 2 macro() calls which are rather expensive.